### PR TITLE
Chapter 5 & Chapter 6

### PR DESCRIPTION
**Pull request by schintees**

### CODE REVIEW

File: src/app/modules/shared/components/presentational/icon-button/icon-button.component.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// @import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
```
Comment (cristinaAnca) [inline src/app/modules/shared/components/presentational/icon-button/icon-button.component.scss]: delete this file if it is not needed

**Review by cristinaAnca — APPROVED** [review]

File: src/app/components/presentational/products-list-view/products-list-view.component.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  tr {
+    &:first-child{
+      font-weight: bold;
+    }
+  }
+  
+  td {
+    padding: 10px 60px 10px 5px;
```
Comment (antonio-marasescu-duran) [inline src/app/components/presentational/products-list-view/products-list-view.component.scss]: **suggestion(non-blocking):** I see your are reutilizing this pixels values in the 'shopping-cart-details-view', you can create a shared `vars.scss`, create some scss variables and import them here.
Another suggestion for this, if the tables are very similar in functionality, you could create a reusable table which takes the headers & values as an input. Now these are just ideas to keep in mind

File: src/app/modules/shared/components/presentational/icon-button/icon-button.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<!-- <button class="btn" [style.color]="color" (click)='onClick.emit()'><i class="fa fa-{{type}}"></i></button> -->
```
Comment (antonio-marasescu-duran) [inline src/app/modules/shared/components/presentational/icon-button/icon-button.component.html]: **chore(non-blocking):** keep in mind to remove commented code and resolve (when the time is right) those todos.

File: src/app/app.component.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+body {
+    font-size: 18px;
```
Comment (antonio-marasescu-duran) [inline src/app/app.component.scss]: **suggestion:** this feels it should be in the global `styles.scss`

**Review by antonio-marasescu-duran — APPROVED** [review]

**Pull request merged**